twitter/app/Http/Controllers/TweetController.php

```diff
@@ -74,4 +83,27 @@ public function delete(int $tweetId): RedirectResponse
         $tweet->deleteTweet($tweetId);
         return redirect()->route('tweets.index');
     }
+
+    /**
+     * ツイートにいいねする
+     */
+    public function favorite(Request $request): JsonResponse
+    {
+        $tweetId = $request->tweetId;
+        $tweet = Tweet::find($tweetId);
+        $userId = Auth::user()->id;
+        if ($tweet->isFavorite($userId)) {
+            $tweet->unfavorite($userId);
+        } else {
+            $tweet->favorite($userId);
+        }
+
+        $tweetFavoritesCount = $tweet->favorites()->count();
+        $isFavoriteTweet = $tweet->isFavorite($userId);
```

Comment:
95行目と101行目って同じメソッドを呼び出してるよね？
であれば、返却された結果を変数に格納して使おう！
```suggestion
        $isFavoriteTweet = $tweet->isFavorite($userId);
        $isFavoriteTweet ? 
            $tweet->unfavorite($userId) :
            $tweet->favorite($userId);

        $tweetFavoritesCount = $tweet->favorites()->count();
```